config/jobs/kubernetes/sig-scalability/sig-scalability-experimental-periodic-jobs.yaml

```diff
@@ -132,6 +132,79 @@ periodics:
           cpu: 2
           memory: "6Gi"
 
+- name: ci-kubernetes-e2e-gci-gce-scalability-nftables-proxy
+  cluster: k8s-infra-prow-build
+  interval: 12h
+  tags:
+    - "perfDashPrefix: nftables"
+    - "perfDashJobType: nftables"
```

Comment:
this has to be performance - those are modes that perfdash understand and you're running the regular scalability tests

Reply:
done